src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,53 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { HTMLAttributes } from "react";
+import { cn } from "../../utilities/index";
+import React from "react";
+
+const badge = cva(
+	"inline-flex h-5 items-center px-2 py-0.5 text-text-xs font-semibold rounded-full",
```

Comment:
I think something is off with the badge in [Figma](https://www.figma.com/file/JsyitDXqF86SbgOINBrnv5/ZenML-Design-System?node-id=736%3A2803&mode=dev) either it's the height, or the font-size my initial guess would be